backend/de.metas.salescandidate.base/src/main/java/de/metas/ordercandidate/api/impl/OLCandEffectiveValuesBL.java

```diff
@@ -356,6 +356,15 @@ public HUPIItemProductId getEffectivePackingInstructions(@NonNull final I_C_OLCa
 		return HUPIItemProductId.ofRepoIdOrNull(olCand.getM_HU_PI_Item_Product_ID());
 	}
 
+	@Override
+	@NonNull
+	public BigDecimal getEffectiveQtyEntered(@NonNull final I_C_OLCand olCand)
+	{
+		return Optional.ofNullable(olCand.getQtyEntered_Override())
```

Comment:
please work with `InterfaceWrapperHelper#isNull` to sitinguish between zero and not-set

Reply:
alright, we'll allow zero, too